Lab2 jonas comments (#12)
clinical-genomics-uppsala/PrecMed1: docs/lab2.md

    | --- | --- | ---| --- | ---|
    | SampleC | 34 | 91 | 34 | 99 | 
    ??? tip

>         Is the reads mapped to the entire genome?

Author: ```suggestion
        Can we map reads to the entire genome?
```

If we compare two variantcallers ([GATK's Haplotypcaller](https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller) and [Google's Deepvariant](https://github.com/google/deepvariant)); both specialized on germline calling, both use the flag `GQ` and the `QUAL` column as a quality measurements, we see a clear difference on both of the quality scores for the same variants. That said, the genotype quality score is not useless, but actually a good quality measurement of a variant call, you just always have to know how the file were produced and what scoring thresholds are considered "good" for that specific program. 

### Background and Artifacts
Another step for validating you variant is eliminating the risk that the variant actually is a sequencing artifact or just background from a "messy" region.  

> To identify artifacts you have to be able to separate the calls that look like a real call (quality, depth etc often look very good), but is actually a result of some sort of introduced bias, from a disease causing variant. What further complicates the process is that since in sequencing we use organic enzymes and proteins we often introduce artifacts in the same region as the cell might have trouble processing the DNA such as long repetitive sequences or homopolymers. Some artifacts are easier to identify, for example sequencing chemistry can affect our ability to sequence certain areas.

Author: ```suggestion
To identify artifacts you have to be able to separate the real variant calls from the variants that look like a real disease causing variant but is actually a call introduced by of some sort of bias. What further complicates the process is that since in sequencing we use organic enzymes and proteins we often introduce artifacts in the same region as the cell might have trouble processing the DNA, such as long repetitive sequences or homopolymers. Some artifacts are easier to identify, for example sequencing chemistry can affect our ability to sequence certain areas.
```
